tests/MongoDB.Driver.Tests/Specifications/crud/CrudOperationTestBase.cs

```diff
@@ -98,7 +98,7 @@ protected sealed override void Execute(IMongoDatabase database, IMongoCollection
 
         protected override void AssertOutcome(BsonDocument outcome, IMongoDatabase database, IMongoCollection<BsonDocument> collection)
         {
-            if (outcome != null && outcome.Contains("result"))
+            if (outcome != null && outcome.Contains("result") && ActualException == null)
```

Comment:
I wondered about this.

If the method throws an exception then there is no result.

Reply:
yes, but I guess we need to update json tests rather than having such modification.

For me, the test description sounds like we expect both things: the `result` node and the exception. But we obviously have only exception.
Maybe we can just quickly create a spec ticket for it? Current code we can leave as is now and update it when spec ticket will be approved?